tests/src/e2e-full.spec.ts

```diff
@@ -0,0 +1,78 @@
+import type { ElectronApplication, Page } from 'playwright';
+import { _electron as electron } from 'playwright';
+import { afterAll, beforeAll, test } from 'vitest';
+import { expect as playExpect } from '@playwright/test';
+import { existsSync } from 'node:fs';
+import { rm } from 'node:fs/promises';
+
+let electronApp: ElectronApplication;
+let page: Page;
+
+beforeAll(async () => {
+  // remove all videos/screenshots
+  if (existsSync('tests/output')) {
+    console.log('Cleaning up output folder...');
+    await rm('tests/output', { recursive: true, force: true });
+  }
+
+  electronApp = await electron.launch({
+    args: ['.'],
+    recordVideo: {
+      dir: 'tests/output/videos',
+      size: {
+        width: 1050,
+        height: 700,
+      },
+    },
+  });
+
+  page = await electronApp.firstWindow();
+
+  page.on('console', console.log);
+
+  const checkLoader = page.getByRole('heading', { name: 'Initializing...' });
+  await playExpect(checkLoader).toHaveCount(0, { timeout: 10000 });
+
+  if ((await page.locator('button:text("Go to Podman Desktop")').count()) > 0) {
+    await page.locator('button:text("Go to Podman Desktop")').click();
+  }
+});
+
+afterAll(async () => {
+  await electronApp.close();
+});
+
+test('Pull and check image', async () => {
+  const images = page.getByRole('link', { name: 'Images' });
+  await playExpect(images).toBeVisible();
+  await images.click();
```

Comment:
it should use a link from the Navigation Bar. (could be a link somewhere else there)

Reply:
also I would rename images to imagesLink or something like that as images may let think that it contains some elements, but it's just a pointer